code_style.sh

```diff
@@ -53,16 +52,18 @@ for name in ${clang_format_exe_names[@]}; do
     break
   fi
 done
-if [ -z "${clang_format_exe}" ]; then
-  echo "error: clang-format not found";
-  exit 1;
-fi
 
 # format C/C++
-clang_format_args="$([ -n "$fix" ] && echo '-i' || echo '--dry-run --Werror')"
-if ! find_cfiles -exec "${clang_format_exe}" $clang_format_args '{}' '+'; then
-  [ -n "$fix" ] || echo 'C/C++ code needs formatting'
-  exitcode=1
+if [ -z "${clang_format_exe}" ]; then
+  echo "error: clang-format-17 not found"
+  # We ignore that error: clang-format-17 is outdated nowadays.
+  #exitcode=1
```

Comment:
In the current state, the CI will silently stop working if we don't install the appropriate binaries. I prefer change to be something like this instead:

```suggestion
  if [ -n "${CI}" ]; then
    exitcode=1
  fi
```